sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -25,7 +25,7 @@ string name = widget.name;
 
 ### Get a JSON property idiomatically
 
-To treat the dynamic content like a standard .NET type, it is recommended to pass `DynamicCaseMapping.PascalToCamel` when calling `ToDynamicFromJson()`.  This enables getting JSON members with PascalCase property names, and will write any JSON members with camelCase names.
+To use dynamic content with an idiomatic .NET style, pass `DynamicCaseMapping.PascalToCamel` to `ToDynamicFromJson()`.  This will allow using PascalCase property names to get camelCase JSON members.
```

Comment:
Don't forget to change this when your other PR goes in re: the naming of this type.